core/src/avm2/globals/flash/events/local_to_stage.rs

```diff
@@ -0,0 +1,66 @@
+use crate::avm2::object::{Object, TObject};
```

Comment:
I don't think it's a good idea to put these definitions in a file in `globals/flash/events/`, since that would normally be reserved for a Rust-defined class called `LocalToStage`- while it doesn't exist, I think these functions should be kept in `mouse_event`.